<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionSave_Screenshot"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuAbout">
     <property name="title">
      <string>About</string>
     </property>
     <addaction name="actionAboutOpenGL"/>
     <addaction name="actionAboutQt"/>
    </widget>
    <addaction name="menuAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionSave_project"/>
   <addaction name="actionSave_Screenshot"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/Folder.png</normaloff>:/Icons/Icons/Folder.png</iconset>
   </property>
   <property name="text">
    <string>Open project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/Disquette.png</normaloff>:/Icons/Icons/Disquette.png</iconset>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/Power.png</normaloff>:/Icons/Icons/Power.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_project">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/Empty document.png</normaloff>:/Icons/Icons/Empty document.png</iconset>
   </property>
   <property name="text">
    <string>New project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAboutOpenGL">
   <property name="text">
    <string>OpenGL</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionSave_Screenshot">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/camera.png</normaloff>:/Icons/Icons/camera.png</iconset>
   </property>
   <property name="text">
    <string>Save screenshot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
